content/departments/cloud/technical-docs/operations.md

```diff
@@ -432,6 +432,59 @@ resource.type="gce_instance"
 protoPayload.authenticationInfo.principalEmail="system@google.com"
 ```
 
+### Enabling GCP Cloud Tracing on Managed Instances
+
+As part of the [Centralised Observability](https://github.com/sourcegraph/customer/issues/1151) efforts we will be enabling GCP Cloud Tracing on all Managed Instances.
+
+To enable GCP Cloud Tracing on an existing instance, follow these instructions for deploying the `opentelemetry-collector` service:
+
+1. `cd` into the desired customer directory and into the deployment folder (`red` or `black`).
+2. Create a new directory called `otel-collector` and within it a folder called `config.yaml` with the following contents:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  jaeger:
+    endpoint: '$JAEGER_HOST:14250'
+    tls:
+      insecure: true
+  googlecloud:
+    retry_on_failure:
+      enabled: false
+
+extensions:
+  health_check:
+    port: 13133
+  zpages:
+    endpoint: 'localhost:55679'
+
+service:
+  extensions: [health_check, zpages]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [jaeger, googlecloud]
+```
+
+3. Update the corresponding `docker-compose.override.yaml` file to now include the `otel-collector` service and configure all existing services with the correct environment variables. Reference this [pull request enabling otel for `tpgi`](https://github.com/sourcegraph/deploy-sourcegraph-managed/pull/911/files) for an example of all that needs to be modified.
```

Comment:
Is this instance name sensitive information?

Reply:
`tpgi` is an internally-used instance for validating accessibility, it's not a customer instance. That being said, I can remove the name since it's not necessary information and could be confusing for others.